Jenkinsfile

```diff
@@ -101,23 +101,18 @@ pipeline {
             label 'docker'
           }
           environment {
-            XMOSDOC_VERSION = "v3.0.0"
+            XMOSDOC_VERSION = "pr-67"
           }
           stages {
             stage('Build Docs') {
               steps {
                 runningOn(env.NODE_NAME)
                 checkout scm
-                sh "docker pull ghcr.io/xmos/doc_builder:${XMOSDOC_VERSION}"
-                sh """docker run --user "\$(id -u):\$(id -g)" \
-                        --rm \
-                        -v ${WORKSPACE}:/build \
-                        -e EXCLUDE_PATTERNS="/build/doc/doc_excludes.txt" \
-                        -e DOXYGEN=1 -e DOXYGEN_INCLUDE=/build/doc/Doxyfile.inc \
-                        -e PDF=1 \
-                        ghcr.io/xmos/doc_builder:${XMOSDOC_VERSION} \
-                        || echo "PDF build is badly broken, ignoring for now till it's fixed." """
-                
+                sh "docker pull ghcr.io/xmos/doc_builder:$XMOSDOC_VERSION"
+                sh """docker run -u "\$(id -u):\$(id -g)" \
+                    --rm \
+                    -v ${WORKSPACE}:/build \
+                    ghcr.io/xmos/doc_builder:$XMOSDOC_VERSION -v"""
```

Comment:
can we zip the doc files so that the artifacts look tidier?

Reply:
We could but then we lose the ability to render them out directly, you'd have to download and open them like with github actions. (eg http://srv-bri-jcim0:8080/job/XMOS/job/lib_xcore_math/job/PR-154/2/artifact/doc/_build/html/index.html)

If you specifically want them in a zip, classic mode has a button for that.

http://srv-bri-jcim0:8080/job/XMOS/job/lib_xcore_math/job/PR-154/2/artifact/doc/_build/

![image](https://github.com/xmos/lib_xcore_math/assets/15315756/32b1d971-81a0-403e-87b6-c074c82c72ce)

Reply:
Thanks for the info, but the Blue Ocean artifacts overview, which most of the developers use, is pretty useless due to the large number of files.

Reply:
I guess there's a discussion to be had which is more valuable.

Theres a blue link at the very top of that page called "Build Artifacts".

![image](https://github.com/xmos/lib_xcore_math/assets/15315756/7cb9e4d5-0470-46cd-89f6-181c16fa24e4)